mmv1/third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -443,7 +443,22 @@ is set to true.`,
 				Computed:    true,
 				Description: `The connection name of the instance to be used in connection strings. For example, when connecting with Cloud SQL Proxy.`,
 			},
-
+                        "maintenance_version": {
+                                Type:        schema.TypeString,
+                                Computed:    true,
+                                Optional:    true,
+                                DiffSuppressFunc: maintenanceVersionDiffSuppress,
+                                Description: `Maintenance version.`,
+                        },
+                        "available_maintenance_versions": {
+                                Type:        schema.TypeList,
+                                Computed:    true,
+                                Optional:    true,
```

Comment:
If this is an output-only field, we probably want to remove `Optional: true`.

Reply:
Done.